shimmingtoolbox/cli/st_mask_square.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import click
+import nibabel as nib
+import numpy as np
+
+from shimmingtoolbox.masking.shapes import shape_square
+
+
+@click.command()
+@click.argument("filename", required=True, type=click.Path(exists=True, file_okay=False))
+@click.option("--len_dim1", '-l1', required=True, type=click.IntRange(0, 255, clamp=True), help="Length of the side "
+                                                                                                "of the square along "
+                                                                                                "third dimension (in "
+                                                                                                "pixels)")
+@click.option("--len_dim2", '-l2', required=True, type=click.IntRange(0, 255, clamp=True), help="Length of the side "
+                                                                                                "of the square along "
+                                                                                                "third dimension (in "
+                                                                                                "pixels)")
+@click.option("--center_dim", nargs=2, default=0, help="Center of the square along first and second dimension (in "
+                                                       "pixels). If no center is provided, the middle is used.")
+def st_mask_cube(filename, len_dim1, len_dim2, center_dim):
+    """
+            Apply a square mask to the input file. Returns mask.
+
+            Args:
+                filename (str): Data to mask.
+                len_dim1 (int): Length of the side of the square along first dimension (in pixels).
+                len_dim2 (int): Length of the side of the square along second dimension (in pixels).
+                center_dim (int): Center of the square along first and second dimension (in pixels). If no center is
+                                provided, the middle is used.
+
+            Return:
+                mask_square (numpy.ndarray): Mask with booleans.
+            """
+    img_nifti = nib.load(filename)
+    data = img_nifti.get_fdata()  # convert nifty file to numpy array
+    mask_square = np.array([[[]]])  # initialization of 3D array
+
+    for z in range(data.shape[2]):
```

Comment:
Square will mostly be used for 2d images, so I would make sure it works for 3d but this will crash for arrays with only 2 dimensions since shape[2] will not exist